collector/test/HostHeuristicsTest.cpp

```diff
@@ -92,10 +77,13 @@ TEST(HostHeuristicsTest, TestARM64Heuristic) {
   MockHostInfoHeuristics host;
   KernelVersion version = KernelVersion("6.5.5-200.fc38.aarch64", "", "aarch64");
   CollectorArgs* args = CollectorArgs::getInstance();
-  MockCollectorConfig config(args);
+  int argc = 3;
+  const char* argv[] = {"collector", "--collection-method", "ebpf"};
```

Comment:
Why is it configured to use `ebpf` everywhere?

Reply:
These tests originally specified ebpf as the collection method. I have not changed that. When we remove ebpf these tests will be changed or removed.

Reply:
> Even if it's better than using a mock, it should be refactored to be more reusable.

I am using mocks now, so this comment is outdated.